docs/reference/presigned-url.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Presigned URL
+description: Configuring lakeFS to use presigned URLs.
+parent: Reference
+nav_order: 100
+has_children: false
+---
+
+# Presigned URL
+{: .no_toc }
+
+{% include toc.html %}
+
+By default, lakeFS will get and set objects using its server. lakeFS also support accessing objects using a presinged URL which allows direct access to the object.
+lakeFS assume by default your object storage support presinged URLs, if not it should be disabled in the configuration.
+
+# Using presigned URLs in the UI
+
+For lakeFS to fetch objects through the UI using a presigned URL (instead of getting it with lakeFS) need to enable the presigned URL support UI in the lakeFS configuration and add cors permissions to the bucket.
```

Comment:
1. Think we need to explain that the UI fetch the objects - its not lakeFS. The lakeFS is the source that the UI can fetch objects from.
2. cors -> CORS (Cross-Origin Resource Sharing) + suggest reference the formal definition

Reply:
There is link on each object store to enable CORS where it is explained